src/driver-roccat-kone-emp.c

```diff
@@ -0,0 +1,1127 @@
+/*
+ * Copyright © 2015 Red Hat, Inc.
```

Comment:
not that I think these statements matter too much, but you may want to add your own copyright here :)